src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -206,6 +206,127 @@ def test_WINNF_FT_S_SIQ_8(self):
     self.assertEqual(response['response']['responseCode'], 103)
 
   @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_10(self):
+    """Send Spectrum Inquiry with Array request with successful and unsuccessful responses.
+
+    The response codes will vary
+    """
+
+    # Register six devices
+    device_1 = json.load(open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_3 = json.load(open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    device_4 = json.load(open(os.path.join('testcases', 'testdata', 'device_f.json')))
+    device_5 = json.load(open(os.path.join('testcases', 'testdata', 'device_g.json')))
+    device_6 = json.load(open(os.path.join('testcases', 'testdata', 'device_i.json')))
+
+    self._sas_admin.InjectFccId({'fccId': device_1['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_2['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_3['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_4['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_5['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_6['fccId']})
+
+    self._sas_admin.InjectUserId({'userId': device_1['userId']})
+    self._sas_admin.InjectUserId({'userId': device_2['userId']})
+    self._sas_admin.InjectUserId({'userId': device_3['userId']})
+    self._sas_admin.InjectUserId({'userId': device_4['userId']})
+    self._sas_admin.InjectUserId({'userId': device_5['userId']})
+    self._sas_admin.InjectUserId({'userId': device_6['userId']})
+
+    # send registration requests
+    devices = [device_1, device_2, device_3, device_4, device_5, device_6]
+    request = {'registrationRequest': devices}
+    response = self._sas.Registration(request)
+
+    # Check registration response
+    cbsd_ids = []
+    for resp in response['registrationResponse']:
+      self.assertEqual(resp['response']['responseCode'], 0)
+      cbsd_ids.append(resp['cbsdId'])
+    del request, response
+
+    # Send Spectrum Inquiry request for six cbsds
+    # create inquiry with 6 requests
+    #
+    # the 1st inquiry is valid
+    spectrum_inquiry_0 = json.load(
```

Comment:
It'll be slightly easier to follow if this is `spectrum_inquiry_1` (matching `device_1`). It's not a big deal (i.e. feel free not to change) but it's nice to do for test cases with many CBSDs.